packages/openactive-integration-tests/test/templates/common.js

```diff
@@ -170,16 +170,19 @@ function additionalDetailsRequiredAndSupplied(req) {
           '@type': 'PropertyValue',
           propertyID: 'https://example.com/experience',
           value: 'I\'ve played twice before, but I\'m a quick learner so I hope to keep up!',
+          name: 'Level of experience',
```

Comment:
Name required: https://openactive.io/open-booking-api/EditorsDraft/1.0CR3/#schema-propertyvaluespecification

Reply:
It shouldn't be required for the `PropertyValue`? Ref: https://github.com/openactive/open-booking-api/issues/188

Reply:
Is the validator throwing up this error?

Reply:
@nickevansuk Yep validator erroring (2.0.184 models anyway)